modules/storage-multi-attach-error.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-storage-issues.adoc
+
+[id="storage-multi-attach-error_{context}"]
+= Resolving multi-attach error issues
+
+.Problem
+When a node crashes or shuts down abruptly, the ReadWriteOnce (RWO) volume is expected to be unmounted from the node so that it can be used by a Pod scheduled on another node. However, this is not possible because when the node is down, the system cannot unmount the volume. A multi-attach error message is reported, for example:
+
+[source,terminal]
+.Example output
+--
+Unable to attach or mount volumes: unmounted volumes=[sso-mysql-pvol], unattached volumes=[sso-mysql-pvol default-token-x4rzc]: timed out waiting for the condition
+Multi-Attach error for volume "pvc-8837384d-69d7-40b2-b2e6-5df86943eef9" Volume is already used by pod(s) sso-mysql-1-ns6b4
+--
+
+.Solution
+.. *Use RWX volumes.*
```

Comment:
I am wondering if bullet points would be more suitable than 'a' and 'b' for the two solutions, because they are not sequentially dependent.

Reply:
It might also be useful to expand this sentence to reflect the intent more fully. For example, 'Use RWX volumes to enable multiple attachments'.